src/Spice86.Core/Emulator/InterruptHandlers/Bios/SystemBiosInt15Handler.cs

```diff
@@ -44,6 +46,40 @@ public class SystemBiosInt15Handler : InterruptHandler {
         Run(operation);
     }
 
+    /// <summary>
+    /// Copy extended memory (XMS), legacy function provided by the BIOS.
+    /// </summary>
+    /// <param name="calledFromVm">Whether the call comes from emulated machine code.</param>
+    public void CopyExtendedMemory(bool calledFromVm) {
+        bool enabled = _memory.A20Gate.IsEnabled;
+        _memory.A20Gate.IsEnabled = true;
+        //https://fd.lod.bz/rbil/interrup/bios/1587.html#table-00499
+        // ES:SI points to descriptor table
+        //  Format of global descriptor table:
+        // Offset	Size	Description	(Table 00499)
+        // ===========================================
+        //  00h 16 BYTEs	zeros (used by BIOS)
+        //  10h	WORD	source segment length in bytes (2*CX-1 or greater)
+        //  12h  3 BYTEs	24-bit linear source address, low byte first
+        //  15h	BYTE	source segment access rights (93h)
+        //  16h	WORD	(286) zero
+        // 		(386+) extended access rights and high byte of source address
+        //  18h	WORD	destination segment length in bytes (2*CX-1 or greater)
+        //  1Ah  3 BYTEs	24-bit linear destination address, low byte first
+        //  1Dh	BYTE	destination segment access rights (93h)
+        //  1Eh	WORD	(286) zero
+        // 		(386+) extended access rights and high byte of destin. address
+        //  20h 16 BYTEs	zeros (used by BIOS to build CS and SS descriptors)
+        uint offset =  MemoryUtils.ToPhysicalAddress(_state.ES, _state.SI);
+        uint sourceAddress = (uint)(_memory.UInt16[offset + 0x12] + _memory.UInt8[offset + 0x16]);
```

Comment:
A few suggestions: 
 - Why not creating a data structure? This way rewritten code would be able to manipulate this easily
 - Almost looks like we need UInt24?
 - Are you sure combining means adding? I would expect to have some bits shifting instead of a +